app_dart/lib/src/request_handlers/check_flaky_builders.dart

```diff
@@ -51,7 +53,15 @@ class CheckFlakyBuilders extends ApiRequestHandler<Body> {
     final GithubService gitHub = config.createGithubServiceWithToken(await config.githubOAuthToken);
     final BigqueryService bigquery = await config.createBigQueryService();
     final String ciContent = await gitHub.getFileContent(slug, kCiYamlPath);
-    final List<_BuilderInfo> eligibleBuilders = await _getEligibleFlakyBuilders(gitHub, slug, content: ciContent);
+    final YamlMap? ci = loadYaml(await gitHub.getFileContent(slug, kCiYamlPath)) as YamlMap?;
+    final pb.SchedulerConfig unCheckedSchedulerConfig = pb.SchedulerConfig()..mergeFromProto3Json(ci);
+    final CiYaml ciYaml = CiYaml(
+      slug: slug,
+      branch: Config.defaultBranch(slug),
+      config: unCheckedSchedulerConfig,
+    );
+    final List<_BuilderInfo> eligibleBuilders =
+        await _getEligibleFlakyBuilders(gitHub, slug, content: ciContent, ciYaml: ciYaml);
```

Comment:
Instead of injecting both ciContent and ciYaml (which are basically the same thing), we can move the ciYaml logic to `_getEligibleFlakyBuilders` and inject only `ciContent`.

Reply:
This will be part of the sweep in the bug that I filed